classes/fakeroot.oeclass

```diff
@@ -3,7 +3,7 @@
 ## By inheriting this class, it is ensures that the native version of fakeroot
 ## build, is used instead of fakeroot from the build system.
 
-CLASS_DEPENDS += "native:fakeroot"
+CLASS_DEPENDS += "native:fakeroot native:libacl1-dev"
```

Comment:
Is this change targeted at native:fakeroot from host distro?  Or native:fakeroot from OE-lite/core recipe?